vpn.py

```diff
@@ -59,8 +63,19 @@ def scan_vpns(directory, auth_file, exclude_list):
     vpn_dir = return_abs_path(directory, "vpns")
     conf_dir = return_abs_path(directory, "configs")
     auth_file = return_abs_path(".", auth_file)
-    for filename in os.listdir(conf_dir):
-        logging.info("Moving onto %s" % (filename))
+    conf_list = os.listdir(conf_dir)
+
+    if shuffle_lists:
+        shuffle(conf_list)
```

Comment:
I don't think this shuffle call makes sense because there is no guaranteed ordering to os.listdir(). It lists directory contents "in arbitrary order" and I don't know exactly what is meant here, but that might not even be consistent between runs. Unless you list the directory and sort the list, you are never guaranteed a consistent order. Does randomization add value here and if you think so, why?